src/components/theme_provider.tsx

```diff
@@ -0,0 +1,40 @@
+import { JSX } from 'react';
+import { ThemeDataType, themes, ThemeType } from 'src/themes';
+import { ThemeProvider } from 'styled-components';
+
+export function AppThemeProvider({
+	children
+}: {
+	children: React.ReactNode;
+}): JSX.Element {
+	const themeName: ThemeType = getThemeName();
+	const themeData: ThemeDataType = getThemeData(themeName);
+	const isEmbedded = themeData.isEmbedded;
+
+	console.log(`Using theme: ${themeName} - isEmbedded: ${isEmbedded}`);
```

Comment:
Remove console.log (or introduce a logger that will get removed for prod builds)

Reply:
Removed - 6aaf85408fe496596b4054f5de21fd43fbd142f7

Reply:
Also renamed this source file - ad8f38c17e9c80f27bb6e055a6163be1c946e93f